azure/src/services/azureFunctionCloudService.ts

```diff
@@ -53,20 +54,10 @@ export class AzureFunctionCloudService implements CloudService {
   public containerResolver: ContainerResolver;
 
   /**
-   *
-   * @param name Name of function to invoke
-   * @param fireAndForget Wait for response if false (default behavior)
-   * @param payload Body of HTTP request
-   * @param headers Headers of the context
-   * @param params StringParams with values for URL
+   * @param invokeOptions invoke interface with parameters needed
    */
-  public async invoke<T>(
-    name: string,
-    fireAndForget,
-    payload: any = null,
-    headers: StringParams = new StringParams(),
-    params: StringParams = new StringParams()
-  ) {
+  public async invoke<T>(invokeOptions: InvokeRequest) {
```

Comment:
This is a breaking change and will require you all to update all `invoke` operations across your code base.

Reply:
Thank you for this feedback. We are aware of this, and we will change all our handlers after this PR is merged